skbio/stats/distance/_base.py

```diff
@@ -487,6 +487,34 @@ def to_data_frame(self):
         """
         return pd.DataFrame(data=self.data, index=self.ids, columns=self.ids)
 
+    @experimental(as_of="0.5.0-dev")
+    def to_series(self):
+        """Create a ``pandas.Series`` from this ``DissimilarityMatrix``.
```

Comment:
Change this line to reference `DistanceMatrix`.

Can you add the following description below this summary line?

```
The series will contain distances in condensed form: only distances from one matrix triangle are included,
and the diagonal is excluded. The series' index will be a ``pd.MultiIndex`` relating pairs of IDs to distances.
The pairs of IDs will be in row-major order with respect to the upper matrix triangle.

To obtain all distances (i.e. both upper and lower matrix triangles and the diagonal), use
``DistanceMatrix.to_data_frame``. To obtain *only* the distances in condensed form
(e.g. for use with SciPy), use ``DistanceMatrix.condensed_form``.
```